src/MovementIndicator.js

```diff
@@ -0,0 +1,18 @@
+const { MOVEMENT_STANDARD_NUMBER } = require("./Utils/Constants");
+
+const randomNumberMaker = () => {
+  return Math.floor(Math.random() * 10);
+};
+
+const isOverFour = (number) => {
+  if (number >= MOVEMENT_STANDARD_NUMBER) {
+    return true;
+  }
+  return false;
+};
```

Comment:
boolean 리턴 함수는 훨씬 간결하게 표현 가능할 것 같네요.
추가적으로 `isOverFour`는 MOVEMENT_STANDARD_NUMBER가 변경됨에 따라 4가 아닐 수도 있기 때문에, isMovementable등으로 치환하면 어떨까요?

```suggestion
const isMovementable= (number) => {
  return number >= MOVEMENT_STANDARD_NUMBER;
};
```

그렇다면 아래의 isMoving 메서드도 제거할 수 있겠네요!